install_area_detection_node:
  ros__parameters:
    depth_topic: "/depth"
    right_topic: "/image_rect"
    camera_info_topic: "/camera_info_rect"
    pose_topic: "/install_area/target_pose"
    vis: true
    shrink_ratio: 0.3
    bracket: # in millimeters
      height: 460
      width: 500
    threshold: # in millimeters
      top: 2500 
      bottom: 1100
